roles/nf-core/templates/site.config.j2

```diff
@@ -0,0 +1,20 @@
+process {
+  {% for i in item.0.containers %}
+  container = '{{ ngi_containers }}/{{ item.0.name }}/{{ i }}'
+  {% endfor %}
```

Comment:
This won't work if there is more than one container, as we will just get `process.container` repeated multiple times. This is essentially just a variable name, so it will overwrite previous instances.

So this will only work when `item.0.containers` happens to be a list with a single element.

For more than that, the nextflow pipeline configuration needs to be more specific about where each container is used.

Reply:
Removed the containers list! We'll let `nf-core download` handle multiple containers.